File: API/Utility/IncomeStatementModel.cs
namespace API.Utility;

public class IncomeStatementResponseModel
{
    public string FromDate { get; set; } = null!;
    public string ToDate { get; set; } = null!;
 
    public decimal RevenuesAmount { get; set; }
    public decimal RevenueReturnsAmount { get; set; }
    public decimal OperatingExpensesAmount { get; set; }
    public decimal AdministrativeExpensesAmount { get; set; }
    public decimal GeneralExpensesAmount { get; set; }
    public decimal SellingExpensesAmount { get; set; }
    public decimal FinancingExpensesAmount { get; set; }
    public decimal CapitalRevenuesAmount { get; set; }
    public decimal AccidentCompensationRevenuesAmount { get; set; }
    
    public decimal NetIncome { get; set; }
  
}

File: client/src/app/income-statement/models/income-statement.model.ts
import IExpensesAccountBalance from "./expenses-account-balance.model";

export default interface IIncomeStatement{
  revenuesAmount: number;
  revenueReturnsAmount: number;
  operatingExpensesAmount: number;
  administrativeExpensesAmount: number;
  generalExpensesAmount: number;
  sellingExpensesAmount: number;
  financingExpensesAmount: number;
  capitalRevenuesAmount: number;
  accidentCompensationRevenuesAmount: number;
  netIncome: number;
  fromDate: string;
  toDate: string;
}
